### PR TITLE
Spring Cleaning

### PR DESCRIPTION
**Pull request by sveinbjornt**
* Got rid of /correct.api endpoint and all code it required
* Deleted lots of unused .conf files and utilities in tools/
* Refactoring and cleanup

### CODE REVIEW

**Comment by sveinbjornt** [comment]
@vthorsteinsson @Holado 

The Greynir repo still contains all sorts of stuff (data in resources/ and config, code in tools/) for evaluating and extending our BÍN vocabulary. It seems to me that this belongs in the BinPackage repo rather than Greynir now that the two have been properly decoupled. Also, some of the helper tools probably belong in GreynirPackage.

What should be thrown away or moved to a different repo? It would be great to get feedback from you guys about this. Candidates for moving or deletion:

```
config/Adjectives.conf
config/FamilyNames.conf
config/Vocab.conf

resources/formers.txt
resources/last.txt
resources/obeygd.hreinsad.txt

tools/compmaker.py
tools/dawgbuilder.py
tools/external_data_validator.py
tools/mim.py
tools/obeygd.py
tools/ptest.py
tools/sortfile.py
tools/speedtest.py
tools/verbs.py
tools/vocab.py
```

**Comment by vthorsteinsson** [comment]
At least resources/formers.txt, resources/last.txt, resources/obeygd.hreinsad.txt can be deleted from this repo, they are now in the BinPackage repo.

Same applies to tools/dawgbuilder.py.

Not sure about the others - would need to look at them individually.

**Comment by sveinbjornt** [comment]
@vthorsteinsson Vil fá grænt ljós á þetta.

File: main.py [hunk 1]
```diff
@@ -216,7 +214,6 @@ def inject_nn_bools() -> Dict[str, Union[str, bool]]:
     )
     # Clobber Settings.DEBUG in GreynirPackage and GreynirCorrect
```
Comment (vthorsteinsson) [inline main.py:215]: Delete reference to GreynirCorrect here

File: tests/test_util.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        "Dóra": "Dora",
+        "Álfur": "Alfur",
+        "GUÐRÚN": "GUDRUN",
+    }
+    for k, v in is2ascii.items():
+        assert icelandic_asciify(k) == v
+
+    from util import read_api_key
+
+    # TODO: Test this function
```
Comment (vthorsteinsson) [inline tests/test_util.py:54]: Something missing here, or intentional? At least the import in line 52 can be commented out if there is no test here, and read_api_key is not called.

**Pull request merged**